src/components/controls/dates/DatePicker/DatePicker.tsx

```diff
@@ -156,18 +157,23 @@ const StyledDatePicker = styled(MantineDatePicker, {
       color: ${({ theme }) => theme.colors.content.tertiary};
     }
   }
-  .mantine-DatePicker-rightSection button {
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    svg {
-      display: none;
-    }
-    &::before {
-      font-family: ${({ theme }) => theme.fonts.body};
-      font-size: ${({ theme }) => theme.fontSizes.md};
-      color: ${({ theme }) => theme.colors.content.primary};
-      content: "❌";
+  .mantine-DatePicker-rightSection {
+    inset-inline-end: ${({ theme }) => 1 * theme.spacingBase}rem;
+    inline-size: ${({ theme }) => 4 * theme.spacingBase}rem;
```

Comment:
By making this 24px, we can make it look exactly like the other components with a right icon:

![image](https://user-images.githubusercontent.com/44197016/212055555-190d229a-0050-483f-a903-49606941f2ad.png)


```suggestion
    inline-size: ${({ theme }) => 3 * theme.spacingBase}rem;
```

Reply:
Good point!